proto/autokitteh/sessions/v1/svc.proto

```diff
@@ -71,10 +71,23 @@ message GetResponse {
 
 message GetLogRequest {
   string session_id = 1 [(buf.validate.field).string.min_len = 1];
+  int32 page_size = 20;
+
+  int32 skip = 21 [(buf.validate.field).cel = {
+    id: "session.list.skip"
+    message: "Must be >= 0"
+    expression: "this >= 0"
+  }];
+
+  string page_token = 22;
+  bool ascending = 11;
```

Comment:
is_accending? in_accending_order?

Reply:
in idiomatic go you shouldn't add is prefix 
and the context is listing, so it is known to be order related